src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -133,6 +134,10 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     ...(isCPS && { pageTitle: `${atiAnalytics.pageTitle} - ${brandName}` }),
   };
 
+  const scrollablePromoVariation = useOptimizelyVariation(
+    'scrollable_promo',
+  ) as unknown as string;
```

Comment:
One thing to check is the problem we had a while ago https://github.com/bbc/simorgh/pull/9891 that @amoore108 fixed, if we can check http://localhost:7080/mundo/23263889 to make sure it doesn't break. If it does I think this can be fixed by using this within the `ScrollablePromo` component, as useOptimizelyVariation hook uses state, it will re-render the whole `ArticlePage` component, but if the hook is within `ScrollablePromo` I think it will only re-render that section of the DOM.

I think in the future we need to think about making a component maybe to hold this hook so state is encapsulated away from the page component

Reply:
good shout, just checked it and all the includes and everything seem to be rendering correctly 👍

Reply:
Yea good shout @HarveyPeachey , had forgotten about that previous behaviour